oddt/fingerprints.py

```diff
@@ -528,6 +530,30 @@ def combinatorial_rmsd(reference, query):
         return np.sqrt((numla / nula) * (numpa / nupa))
 
 
+def PLECFP(ligand, protein, distance_cutoff=4.5, depth_protein=4, depth_ligand=2, fold_size=16384):
```

Comment:
Few things missing or to fix:

- `fold_size` is named `size` elsewhere
- docstrings missing
- it will probably not  survive if you swap the order of protein/ligand mols